apps/api/src/database/migrations/1692870736646-seeddata.ts

```diff
@@ -0,0 +1,51 @@
+import { MigrationInterface, QueryRunner } from 'typeorm';
+
+export class SeedData1692870736646 implements MigrationInterface {
+  name = 'SeedData1692870736646';
+
+  public async up(queryRunner: QueryRunner): Promise<void> {
+    // Insert sample data into a table
+    await queryRunner.query(
+      `INSERT INTO notify_users (username,password,role,created_on,updated_on,status) VALUES ('Admin','$2b$10$iUUsgPtfqu./C2fnnb80EOlNxc3q73woJd2.Ns0D66xHh0iX4E1vq',1,'2024-04-19 13:01:11','2024-04-19 15:37:16',1)`,
+    );
+
+    await queryRunner.query(
+      `INSERT INTO notify_applications (application_id,name,user_id,created_on,updated_on,status) VALUES (1001,'sampleOsmoXApp',1,'2024-04-12 11:36:02','2024-04-12 11:36:02',1)`,
+    );
+
+    await queryRunner.query(
+      `INSERT INTO notify_server_api_keys (api_key,application_id,created_on,updated_on,status) VALUES ('OsmoX-test-key',1001,'2024-04-12 12:28:06','2024-04-12 12:28:06',1)`,
+    );
+
+    await queryRunner.query(
+      `INSERT INTO notify_master_providers (name,provider_type,configuration,created_on,updated_on,status) VALUES ('SMTP',1,'{"ENABLE_SMTP":{"label":"ENABLE SMTP","id":"ENABLE_SMTP","pattern":"/^(true|false)$/","type":"boolean"},"SMTP_HOST":{"label":"SMTP HOST","id":"SMTP_HOST","pattern":"^https?://(?:[^/]+|(?:\\d{1,3}\\.){3}\\d{1,3})/","type":"string"},"SMTP_PORT":{"label":"SMTP PORT","id":"SMTP_PORT","pattern":"\\d{1,5}","type":"number"},"SMTP_USERNAME":{"label":"SMTP USERNAME","id":"SMTP_USERNAME","pattern":"^[a-zA-Z0-9_-]{3,16}$","type":"string"},"SMTP_PASSWORD":{"label":"SMTP PASSWORD","id":"SMTP_PASSWORD","pattern":"^[a-fA-F0-9]{64}$","type":"string"}}','2024-04-24 15:20:23','2024-04-24 15:20:23',1),
```

Comment:
For SMTP_HOST URL, you can use the pattern from here: https://github.com/OsmosysSoftware/dev-standards/blob/main/coding-standards/RegEx.md if it is fine.
Also I feel like SMTP_USERNAME upperlimit is too low.

Reply:
Fixed this, upper limit moved to 30